downstream/assemblies/platform/assembly-ag-controller-clustering.adoc

```diff
@@ -0,0 +1,40 @@
+[id="controller-clustering"]
+
+= Clustering
+
+Clustering is sharing load between hosts. 
+Each instance must be able to act as an entry point for UI and API access. 
+This must enable the {ControllerName} administrators to use load balancers in front of as many instances as they wish and maintain good data visibility.
+
+[NOTE]
+====
+Load balancing is optional and is entirely possible to have ingress on one or all instances as needed.
+====
+
+Each instance must be able to join the {ControllerName} cluster and expand its ability to execute jobs. 
+This is a simple system where jobs can run anywhere rather than be directed on where to run. 
+Also, clustered instances can be grouped into different pools or queues, called [ADD XREF]Instance Groups.
+
+{PlatformNameShort} supports container-based clusters using Kubernetes, meaning new controller instances can be installed on this platform without any variation or diversion in functionality. 
+You can create instance groups to point to a Kubernetes container. 
+For more information, see the [ADD XREF]Container and Instance Groups section.
```

Comment:
Is 'Instance Groups' also an xref?

Reply:
No it is the same heading in chapter 8:
https://docs.ansible.com/automation-controller/latest/html/administration/containers_instance_groups.html